setup.py

```diff
@@ -37,3 +25,18 @@ def write_version_py(filename=None):
 
 if write_version:
     write_version_py()
+
+
+setup(name='geoutils',
+      version=FULLVERSION,
+      description='',
+      url='',
+      author='The GlacioHack Team',
+      license='BSD-3',
+      packages=['geoutils'],
+      install_requires=['rasterio', 'geopandas', 'pyproj'],
```

Comment:
This should be consistent with the requirement.txt file, no?

Reply:
Not necessarily: https://packaging.python.org/discussions/install-requires-vs-requirements/